docs/customer-360/aviation/process.sql

```diff
@@ -0,0 +1,86 @@
+SET 'auto.offset.reset' = 'earliest';
+
+CREATE TABLE customers (ID             INT     PRIMARY KEY
+                       , NAME           VARCHAR
```

Comment:
Formatting seems broken?

Reply:
Formatting concerns are covered as part of https://github.com/confluentinc/ksqldb-recipes/issues/90